src/test/java/com/nerodesk/om/aws/AwsDocsTest.java

```diff
@@ -29,17 +29,39 @@
  */
 package com.nerodesk.om.aws;
 
+import com.google.common.collect.Lists;
+import com.jcabi.s3.Bucket;
+import com.jcabi.s3.Ocket;
+import com.nerodesk.om.Docs;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
 import nl.jqno.equalsverifier.EqualsVerifier;
 import nl.jqno.equalsverifier.Warning;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
 import org.junit.Test;
+import org.mockito.Mockito;
 
 /**
  * Tests for {@link AwsDocs}.
  *
  * @author Krzysztof Krason (Krzysztof.Krason@gmail.com)
  * @version $Id$
+ * @since 0.3
  */
 public final class AwsDocsTest {
+
+    /**
+     * Doc that exists.
+     */
+    private static final String EXISTS = "EXISTS";
```

Comment:
avoid using static fields in tests. It's better to have different values in different places